lib/File/Temp.pm

```diff
@@ -1,7 +1,7 @@
 package File::Temp;
 # ABSTRACT: return name and handle of a temporary file safely
 
-our $VERSION = '0.2312';
+our $VERSION = '0.2313';
```

Comment:
The versions in this dist are managed by the release process. This change should be removed.

Reply:
Thanks, that's done.